strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/ArtifactDownloadingEventHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.carlspring.strongbox.providers.repository;
+
+import java.io.IOException;
+import java.util.Date;
+
+import org.carlspring.strongbox.artifact.AsyncArtifctEntryHandler;
+import org.carlspring.strongbox.domain.ArtifactEntry;
+import org.carlspring.strongbox.event.artifact.ArtifactEventTypeEnum;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ArtifactDownloadingEventHandler extends AsyncArtifctEntryHandler
```

Comment:
There's a typo in the name of the class you're extending -- you're missing an "a".

Reply:
yeah, thanks
fixed